test/common/util.js

```diff
@@ -39,7 +39,11 @@ function getName (file) {
     );
 }
 
+<<<<<<< HEAD
```

Comment:
!!!

Reply:
There was a problem with `code` but it was fixed in a commit before the merge.